main/udev-init-scripts/APKBUILD

```diff
@@ -1,53 +1,33 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=udev-init-scripts
-pkgver=30
-pkgrel=6
+pkgver=32
+pkgrel=0
 pkgdesc="udev startup scripts for openrc"
 url="https://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/sys-fs/udev-init-scripts/"
 arch="noarch"
 license="GPLv2"
-depends=""
-depends_dev=""
-makedepends="$depends_dev"
-replaces="udev"
+provides=udev
```

Comment:
If we do this, it can't co-exist with 'udev' dummy package which is sometimes used to pull in eudev.
@ncopa Should we just nuke 'udev' transitional package, and include provides here and in 'eudev' ?